e2e/test/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -1051,6 +1052,29 @@ describe("scenarios > visualizations > pivot tables", { tags: "@slow" }, () => {
       });
     });
   });
+
+  it("should not have to wait for data to show fields in summarisation (#26467)", () => {
+    createAndVisitTestQuestion();
+
+    cy.intercept("POST", "api/card/pivot/*/query", req => {
+      req.on("response", res => {
+        res.setDelay(20_000);
```

Comment:
```suggestion
        res.setDelay(20000);
```

Reply:
isn't `20_000` more readable? I added the `_` after I realised I had written `200000` by mistake

Reply:
True :+1: